stable/vpa/README.md

```diff
@@ -6,7 +6,7 @@ This chart is entirely based on the manifests and various scripts in the `deploy
 
 ## Tests and Debugging
 
-There are a few tests included with this chart that can help debug why your installation of VPA isn't working as expected. You can run `helm -n <Release Namespace> <Release Name>` to run them.
+There are a few tests included with this chart that can help debug why your installation of VPA isn't working as expected. You can run `helm test -n <Release Namespace> <Release Name>` to run them.
```

Comment:
Premature approval, sorry.

You'll want to add this to Readme.md.gotmpl and then use https://github.com/norwoodj/helm-docs to regenerate the README. Let me know if you need anything for that.

Reply:
Sorry about that.  I think it's corrected now.  After running helm-docs I do see a difference below ('Chart Values' to 'Values') that doesn't seem to be caused by me... perhaps caused by a different version of helm-docs?

Reply:
I'll get this merged https://github.com/FairwindsOps/charts/pull/371 and that should take care of it.